src/js/services/lang.js

```diff
@@ -1,26 +1,24 @@
 /**
- * JS related to our language switcher in the footer
+ * JS related to the language selector in the header nav
  */
 
 // helpers -------------------
 
-function getElById(id) {
-  return document.getElementById(id);
-}
+const getElById = (id) => document.getElementById(id);
 
-var languages = {
+const languages = {
   en: { name: 'English', url: 'docs/' },
   ja: { name: '日本語', url: 'docs/ja/' },
 };
 
 // List of elements that have dynamic content based on our selected language
-var els = {
-  sidebarLangSelect: getElById('sidebarLangSelect'),
-  footerLangSelect: getElById('footerLangSelect'),
-  footerLangOptions: getElById('footerLangOptions'),
+const els = {
```

Comment:
I think we have opportunity here to change the name 'els'. I think it means english-language-selector but it looks really close to else. I believe in jekyll there is an `elsif`. I am not 100% on this suggestion. I just wanted to point it out.

Reply:
Ah I think Tyler suggestion below fixes this.

Reply:
Good call out @ccgeller  - I originally wrote that code and `els` refers to `elements` (dom elements).

Reply:
will update!